source/motor-async-migration.txt

```diff
@@ -0,0 +1,74 @@
+.. _pymongo-async-motor-migration:
+
+====================================
+Migrate from Motor to {+driver-async+}
+====================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: motor, async, refactor, migration
+
+Overview
+--------
+
+The {+driver-async+} driver is a unification of {+driver-short+} and the `Motor
+library <https://www.mongodb.com/docs/drivers/motor/>`__. In this guide, you can
+identify the changes you must make to migrate an application from
+Motor to the {+driver-async+} driver.
+
+Migrate From Motor
+------------------
+
+The {+driver-async+} driver functions similarly to the Motor library, but allows
+for improved latency and throughput due to directly using Python Asyncio instead
+of delegating work to a thread pool. In most cases, you can directly migrate
+existing Motor applications to {+driver-async+} by using ``AsyncMongoClient`` in
+place of ``MotorClient``, and changing the application's import statements to
+import from ``pymongo``.
+
+The following example shows the difference in imports to use a client for
+read and write operations in Motor compared to {+driver-async+}:
+
+.. code-block:: python
+
+   # Motor client import
+   from motor.motor_asyncio import AsyncIOMotorClient
+
+   # {+driver-async+} client import
+   from pymongo import AsyncMongoClient
+
+To see a list of the asynchronous methods available in the {+driver-async+}
+driver, see the :ref:`pymongo-async-methods` section in the {+driver-short+} to
+{+driver-async+} guide.
+
+The following section shows the method signature changes that you must implement
+in your application when migrating from Motor to the {+driver-async+} driver.
+
+Method Signature Changes
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following Motor method signatures behave differently in the {+driver-async+} driver:
+
+- ``GridOut.open()`` returns ``None``.
```

Comment:
Does `GridOut` here refer to a motor class? The pymongo class is `AsyncGridOut`.